ajax/libs/deferer/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "deferer",
+  "version": "1.0.0",
+  "description": "Redirect traffic for matching referering hosts",
+  "author": "Dominic Dagradi <ddagradi@gmail.com>",
+  "repository": "https://github.com/dominic/deferer",
+  "main": "deferer.js",
+  "license": "MIT",
+  "dependencies": {},
+  "npmName": "deferer",
+  "npmFileMap": [
+    {
+      "basePath": "/",
+      "files": [
+        "*/"
```

Comment:
@dominic @PeterDaveHello The cdnjs-importer searches for a directory like `src`, `dist` or `lib` -- if nothing is found, then it fallsback to the root directory. Any idea how can we improve that?